os/arch/arm/src/amebasmart/amebasmart_pmhelpers.c

```diff
@@ -132,9 +133,24 @@ void pg_timer_int_handler(void *Data)
 	// Reset the global struct
 	g_pm_timer.timer_type = 0;
 	g_pm_timer.timer_interval = 0;
+	g_pm_timer.last_wakeup_time = 0;
 }
 
 void up_set_pm_timer(void) {
+
```

Comment:
This is a pm timer for BSP. But what you added is a pm general functionality. Are you able to move them into the pm? not bsp?

Reply:
Yes correct , I moved it to PM .